execution/scan/literal_selector.go

```diff
@@ -83,6 +100,8 @@ func (o *numberLiteralSelector) Next(ctx context.Context) ([]model.StepVector, e
 		o.step = 1
 	}
 	o.currentStep += o.step * int64(o.numSteps)
+	duration := time.Since(start)
+	o.AddCPUTimeTaken(duration)
```

Comment:
```suggestion
	o.AddCPUTimeTaken(time.Since(start))
```

We typically avoid creating variables that are used on the next line or if they are only used in one place.